src/app/routes/index.test.jsx

```diff
@@ -25,9 +25,18 @@ import storyPageMostReadData from '#data/pidgin/mostRead/index.json';
 
 fetchMock.config.fallbackToNetwork = true; // ensures non mocked requests fallback to an actual network request
 
+beforeEach(() => {
+  // Mocks out CanonicalAdBootstrapJs script
```

Comment:
May I ask why do we need to mock it also at this level?

Reply:
Part of it is testing FrontPage/index.jsx which calls this bootstrap script. But the script is pushed to the header which doesn't run within unit tests. I wonder if these tests are more like Integration tests than unit tests?